embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/capture/crumbs/EmbraceBreadcrumbService.kt

```diff
@@ -455,13 +313,13 @@ internal class EmbraceBreadcrumbService(
 
     override fun cleanCollections() {
         viewBreadcrumbs.clear()
```

Comment:
Will these be moved as well later or are they staying in this service for a reason?

Reply:
These are a little bit harder to grok - breadcrumbs are removed from the queue after being added & IMO they should probably be modelled as spans instead. I'll extract these in a future PR.